shelley/chain-and-ledger/executable-spec/test/Rules/ClassifyTraces.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Rules.ClassifyTraces where
+
+import           Data.Foldable (toList)
+import qualified Data.Set as Set
+import           Hedgehog (Property, cover, forAll, property, withTests)
+
+import           Control.State.Transition.Generator (traceOfLength)
+import           Control.State.Transition.Trace (TraceOrder (OldestFirst), traceLength,
+                     traceSignals)
+
+import           Delegation.Certificates (pattern RegKey)
+import           Generator.LedgerTrace ()
+import           MockTypes (DCert, LEDGER, Tx)
+import           TxData (_body, _certs, _inputs, _outputs)
+
+relevantCasesAreCovered :: Property
+relevantCasesAreCovered = withTests 100 $ property $ do
```

Comment:
Don't we want to test with more than 100 to have more stability in the coverage metrics?

Reply:
👍